src/replication.thrift

```diff
@@ -890,6 +890,38 @@ struct start_partition_split_response
     2:string            hint_msg;
 }
 
+enum split_control_type
+{
+    PSC_PAUSE,
```

Comment:
Now that the enums are defined in shrift, mean we will use them as `split_control_type:: PSC_PAUSE ` , is it needed to add the confusing prefix `PSC`？

Reply:
PSC is short for partition split control, actually it is redundant, I will remove it.